src/Schema/Events/Tap.ts

```diff
@@ -139,6 +139,29 @@ export interface TappedAuctionGroup extends TappedEntityGroup {
   action: ActionType.tappedAuctionGroup
 }
 
+/**
+ * A user taps a grouping of auction results on iOS
+ *
+ * This schema describes events sent to Segment from [[tappedEntityGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedAuctionResultGroup",
+ *    context_module: "auctionResults",
+ *    context_screen_owner_type: "artistInsights",
+ *    context_screen_owner_id: "51aa03df8b3b8177260002ab",
+ *    context_screen_owner_slug: "nicolas-party",
+ *    destination_screen_owner_type: "auctionResult",
+ *    destination_screen_owner_id: "6398282",
```

Comment:
This assumes that an auction result screen has an auction result id in `diffusion.lots`.